src/config.ts

```diff
@@ -43,7 +43,7 @@ const Config = {
       draft: true, // Mark it as a draft
       prerelease: false, // Mark it as a prerelease
       files: [], // Globs of files to attach to the release
-      token: '' // GitHub OAuth token with `public_repo` priviledge
+      token: ('' || process.env.GITHUB_TOKEN) // GitHub OAuth token with `public_repo` priviledge
```

Comment:
This line doesn't make sense. `''` will never be truthy, and env variables can be undefined, while we expect a string here.

Reply:
Really no make a sense, but the Javascript no make sense for me. LOL.